packages/repository-json-schema/src/build-schema.ts

```diff
@@ -15,6 +15,7 @@ import {
 import * as debugFactory from 'debug';
 import {JSONSchema6 as JSONSchema} from 'json-schema';
 import {JSON_SCHEMA_KEY, MODEL_TYPE_KEYS} from './keys';
+const stringifyObject = require('stringify-object');
```

Comment:
Can we simply use https://nodejs.org/api/util.html#util_util_inspect_object_options?

Reply:
Ok, tried and it works smooth. I put these options:
```
{
	"depth": Infinity,
	"maxArrayLength": Infinity,
	"breakLength": Infinity
}
```
The style is slightly different but is readable too. i.e:
...(Schema options: { optional: [ 'id' ] })